### PR TITLE
check for nil app endpoints

### PR DESCRIPTION
**Pull request by danvixent**

### CODE REVIEW

**Review by jirevwe — APPROVED** [review]

**Review by Dotunj — APPROVED** [review]

File: datastore/badger/application.go [hunk 1]
```diff
@@ -66,12 +66,17 @@ func (a *appRepo) LoadApplicationsPaged(ctx context.Context, gid, q string, page
 		return nil, datastore.PaginationData{}, err
 	}
 
-	for i, ap := range apps {
+	for i := range apps {
```
Comment (oneEyedSunday) [inline datastore/badger/application.go:69]: If the ap.Endpoints is nil, can we skip the a.db.Count query since its expected to be zero?

Comment (danvixent) [reply datastore/badger/application.go:69]: I think that would be an incorrect assumption, what if the endpoints were deleted after events have been pushed through the app?

Comment (oneEyedSunday) [reply datastore/badger/application.go:69]: I guess my question is, is it correct to assume that if there are no application endpoints, there are no application events?

Comment (danvixent) [reply datastore/badger/application.go:69]: No i don't think so. Deletions of either may have occured.

**Comment by oneEyedSunday** [comment]
Ok

**Pull request merged**